R/fcast.R

```diff
@@ -19,7 +19,7 @@ dcast <- function(data, formula, fun.aggregate = NULL, ..., margins = NULL,
 
 check_formula <- function(formula, varnames, valnames) {
   if (is.character(formula)) formula = as.formula(formula)
-  if (class(formula) != "formula" || length(formula) != 3L)
+  if (!inherits(formula, "formula") || length(formula) != 3L)
```

Comment:
Nice find.   I was surprised to find `is.formula()` doesn't exist.  So `inherits()` it is.